build.gradle

```diff
@@ -72,17 +65,17 @@ subprojects {
     }
 
     dependencies {
-        implementation(platform("com.fasterxml.jackson:jackson-bom:${jacksonVersion}"))
+        implementation(platform(libs.jackson.bom))
         implementation("com.fasterxml.jackson.core:jackson-annotations")
-        implementation("com.google.guava:guava:33.0.0-jre")
-        implementation("org.jetbrains:annotations:24.0.0")
-        implementation("org.slf4j:slf4j-api:2.0.12")
-        compileOnly("com.github.spotbugs:spotbugs-annotations:4.8.5")
+        implementation(libs.guava)
+        implementation(libs.slf4j.api)
+        compileOnly(libs.jetbrains.annotations)
+        compileOnly(libs.spotbugs.annotations)
 
-        testImplementation(platform("org.junit:junit-bom:5.10.3"))
+        testImplementation(platform(libs.junit.bom))
         testImplementation("org.junit.jupiter:junit-jupiter")
```

Comment:
Can we use `lib.xxx` here as well?

Reply:
We could - it's a taste/preference thing.
There can be a lot of "unmanaged" dependencies (version-managed via a bom) mentioned in `libs.gradle.toml` over time.

Reply:
WRT to "visible dependency report" - I'll come up w/ additional PRs to a) automatically licenses, b) verify `NOTICE` content and c) generate a license report containing all deps in distributables.

Reply:
> it's a taste/preference thing.

I'd prefer a consistent style, but we can fix it in followup PRs.